packages/maskbook/src/plugins/Trader/apis/trending/index.ts

```diff
@@ -9,6 +9,8 @@ import { resolveCoinId, resolveCoinAddress, resolveAlias } from './hotfix'
 import STOCKS_KEYWORDS from './stocks.json'
 import CASHTAG_KEYWORDS from './cashtag.json'
 import HASHTAG_KEYWORDS from './hashtag.json'
+import PREDICTION_MARKET_TOKENS from './prediction_market_tokens.json'
```

Comment:
Please move this file to the `apis/unswap/index.ts`.

Reply:
I don't think so, PMTK may will have other data provider.

Reply:
But you treat those tokens owned by Uniswap provider. So it's better to encapsulate to `apis/uniswap`.

Reply:
ok